docs/pages/user-manual.mdx

```diff
@@ -107,77 +106,68 @@ $ tsh login --proxy=work.example.com:2002
 
 <Admonition
   type="tip"
   title="Tip"
 >
-  It is recommended to always use [`tsh login`](cli-docs.mdx#tsh-login) before
-  using any other `tsh` commands. This
-  allows users to omit `--proxy` flag in subsequent tsh commands. For example
+  It is recommended to always use [`tsh login`](cli-docs.mdx#tsh-login) before using any other `tsh` commands. This allows users to omit `--proxy` flag in subsequent tsh commands. For example
   `tsh ssh user@host` will work.
 </Admonition>
 
-A Teleport cluster can be configured for multiple user identity sources. For
-example, a cluster may have a local user called `admin` while regular users
-should [authenticate via Github](admin-guide.mdx#github-oauth-20). In this case,
-you have to pass `--auth` flag to `tsh login` to specify which identity storage
-to use:
+A Teleport cluster can be configured for multiple user identity sources. For example, a cluster may have a local user called `admin` while regular users should [authenticate via Github](admin-guide.mdx#github-oauth-20). In this case, you have to pass `--auth` flag to `tsh login` to specify which identity storage to use:
 
 ```bash
 # Login using the local Teleport 'admin' user:
-$ tsh --proxy=proxy.example.com --auth=local --user=admin login
+tsh --proxy=proxy.example.com --auth=local --user=admin login
 
 # Login using Github as an SSO provider, assuming the Github connector is called "github"
-$ tsh --proxy=proxy.example.com --auth=github --user=admin login
+tsh --proxy=proxy.example.com --auth=github --user=admin login
 ```
 
 When using an external identity provider to log in, `tsh` will need to open a web browser to
 complete the authentication flow. By default, `tsh` will use your system's default browser to open
-such links. If you wish to suppress this behaviour, you can use the `--browser=none` flag:
+such links. If you wish to suppress this behavior, you can use the `--browser=none` flag:
 
 ```
 # Don't open the system default browser when logging in
-$ tsh login --proxy=work.example.com --browser=none
+tsh login --proxy=work.example.com --browser=none
 ```
 
-In this situation, a link will be printed to the screen. You can copy and paste this link into
+In this situation, a link will be printed on the screen. You can copy and paste this link into
 a browser of your choice to continue the login flow.
 
 [CLI Docs - tsh login](cli-docs.mdx#tsh-login)
 
-### Inspecting SSH Certificate
+### Inspecting SSH certificate
 
 To inspect the SSH certificates in `~/.tsh`, a user may execute the following
 command:
 
 ```bash
-$ tsh status
+tsh status
 
-> Profile URL:  https://proxy.example.com:3080
-  Logged in as: johndoe
-  Roles:        admin*
-  Logins:       root, admin, guest
-  Valid until:  2017-04-25 15:02:30 -0700 PDT [valid for 1h0m0s]
-  Extensions:   permit-agent-forwarding, permit-port-forwarding, permit-pty
+# > Profile URL:  https://proxy.example.com:3080
```

Comment:
Since this, we've added Cluster to the `tsh status` output

```
➜  ~ tsh status
> Profile URL:        https://teleport.asteroid.earth:443
  Logged in as:       benarent
  Cluster:            teleport.asteroid.earth
  Roles:              access, editor
  Logins:             benarent
  Kubernetes:         enabled
  Kubernetes cluster: "floral-creature-1621961244"
  Valid until:        2021-05-26 04:37:20 -0700 PDT [valid for 12h0m0s]
  Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty
```

Reply:
Corrected, thanks!